registry/raw.go

```diff
@@ -1,222 +1,21 @@
 package registry
 
-import (
-	"github.com/mongodb/amboy"
-	"github.com/pkg/errors"
-	legacyBSON "gopkg.in/mgo.v2/bson"
-)
+import mgobson "gopkg.in/mgo.v2/bson"
 
-type rawJob struct {
-	Body []byte `bson:"body" json:"body" yaml:"body"`
-	Type string `bson:"type" json:"type" yaml:"type"`
-	job  interface{}
-}
+type rawJob []byte
 
-func (j *rawJob) SetBSON(r legacyBSON.Raw) error { j.Body = r.Data; return nil }
-func (j *rawJob) GetBSON() (interface{}, error) { // Get ~= Marshal
-	if j.job != nil {
-		return j.job, nil
-	}
+func (j rawJob) MarshalJSON() ([]byte, error)   { return j, nil }
```

Comment:
This can be a pointer... the json library, unlike the bson library, works with nicely with converting values to pointers.